lib/smart_app_launch/client_suite/token_request_bsca_verification_test.rb

```diff
@@ -17,7 +17,16 @@ class SMARTClientTokenRequestBackendServicesConfidentialAsymmetricVerification <
     title 'Verify SMART Token Requests'
     description %(
         Check that SMART token requests are conformant.
-      )
+    )
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@229',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@230',
```

Comment:
I don't see @230 tested here. Please confirm and remove if you agree.

@230
To begin the exchange, the client SHALL use the [Transport Layer Security (TLS) Protocol Version 1.2 (RFC5246)](https://tools.ietf.org/html/rfc5246) or a more recent version of TLS to authenticate the identity of the FHIR authorization server and to establish an encrypted, integrity-protected link for securing all exchanges between the client and the FHIR authorization server’s token endpoint.